addons/mail/models/mail_channel_rtc_session.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+
+
+class MailRtcSession(models.Model):
+    _name = 'mail.channel.rtc.session'
+    _description = 'Mail RTC session'
+
+    channel_id = fields.Many2one('mail.channel', index=True, required=True, ondelete='cascade')
```

Comment:
should be related to `channel_partner_id.channel_id`, maybe with store=True if you need